subscription_graphql_ws_test.go

```diff
@@ -567,3 +567,91 @@ func TestSubscription_closeThenRun(t *testing.T) {
 		t.Fatalf("got error: %v, want: nil", err)
 	}
 }
+
+// waitForConnectionState waits for the subscription client to reach a specific connection state
+func waitForConnectionState(t *testing.T, sc *SubscriptionClient, tickerDuration time.Duration, timeout time.Duration, checkFn func() bool) bool {
```

Comment:
utility method to avoid waiting for an established amount and delaying test execution.